### PR TITLE
Register new package SpatialEcology.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [EcoJulia/SpatialEcology.jl](https://github.com/EcoJulia/SpatialEcology.jl)
Release: [v0.1.0](https://github.com/EcoJulia/SpatialEcology.jl/releases/tag/v0.1.0)
cc: @mkborregaard

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@mkborregaard This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

File: SpatialEcology/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.6-pre
+
+RecipesBase
+DataFrames
+Reexport
+PlotUtils
+MacroTools
```
Comment (tkelman) [inline SpatialEcology/versions/0.1.0/requires]: `@capture` didn't exist prior to 0.3.3

Comment (mkborregaard) [inline SpatialEcology/versions/0.1.0/requires]: Nice, thanks for catching this. I reworked the code bit to lose the MacroTools dependency altogether. Best approach to close this and release a 0.1.1?

Comment (tkelman) [inline SpatialEcology/versions/0.1.0/requires]: or replace the tag in place, whichever you prefer (ref https://github.com/attobot/attobot#updatingdeleting-a-release)

**Comment by mkborregaard** [comment]
Thanks, @tkelman !

**Pull request merged**